### ISSUE TITLE
M-x org-publish-o-blog  not working

### ISSUE THREAD

**Issue by AaronFeldman** [renard/o-blog#160]
o-blog on GNU Emacs 24.5.1 (x86_64-unknown-linux-gnu, GTK+ Version 3.10.8) and o-blog using the el-get method for Emacs greater than 24 o-blog installs, updates, and reinstalls, but M-x org-publish-o-blog when using the sample .org file results in "[no-match]".

My underlying system is Ubuntu 14.04, when I installed o-blog, the "o-blog" directory is within the directory "~/.emacs.d/el-get/o-blog", which means that it is not in  "~/.emacs.d/o-blog".  Could that be the source of this error?  

When I attempt publishing the blog using "M-x o-blog-publish" I get this error "byte-code: Symbol's function definition is void: eieio-defclass"

**Comment by cviebig** [comment]
I had issues with their example too but was able to construct a working one. You can have a look on it here: https://github.com/cviebig/o-blog-example Might this solve your issue?

Best
Christoph

**Comment by AaronFeldman** [comment]
Thanks so much.  You know the website example given by Renard seemed so cool that I was looking forward to doing making my own o-blog.  But after struggling with it, and realizing that even the command for publishing had been changed without a change in the documentation on what seemed to be the o-blog install page (but then I found another install page with another "publish" command), and taking steps to make sure v2 was being used, and unsuccessfully attempting the debugging recommended by the author (the "debugging steps" just refused to work, clean environment and all),  and not being a programmer myself, I give up/gave up.  It should not be this hard.
